common/netjack_packet.c

```diff
@@ -1006,7 +1005,7 @@ render_payload_to_jack_ports_16bit (void *packet_payload, jack_nframes_t net_per
         jack_port_t *port = (jack_port_t *) node->data;
         jack_default_audio_sample_t* buf = jack_port_get_buffer (port, nframes);
 
-        float *floatbuf = alloca (sizeof(float) * net_period_down);
+        float floatbuf[sizeof(float) * net_period_down];
```

Comment:
this is wrong, alloca used the full size because it is how it works, but for VLAs the type of the variable is already defined (float in this case) so we need to remove the `sizeof(float) *`part here